modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -46,44 +55,52 @@ $ oc netobserv flows [<feature_option>] [<command_options>]
+|--background|                run in background                          | false
+|--copy|                      copy the output files locally              | prompt
+|--log-level|                 components logs                            | info
+|--max-time|                  maximum capture time                       | 5m
+|--max-bytes|                 maximum capture bytes                      | 50000000 = 50MB
+|--action|                    filter action                              | Accept
+|--cidr|                      filter CIDR                                | 0.0.0.0/0
+|--direction|                 filter direction                           | n/a
+|--dport|                     filter destination port                    | n/a
+|--dport_range|               filter destination port range              | n/a
+|--dports|                    filter on either of two destination ports  | n/a
+|--drops|                     filter flows with only dropped packets     | false
+|--icmp_code|                 filter ICMP code                           | n/a
+|--icmp_type|                 filter ICMP type                           | n/a
+|--node-selector|             capture on specific nodes                  | n/a
+|--peer_ip|                   filter peer IP                             | n/a
+|--peer_cidr|                 filter peer CIDR                           | n/a
+|--port_range|                filter port range                          | n/a
+|--port|                      filter port                                | n/a
+|--ports|                     filter on either of two ports              | n/a
+|--protocol|                  filter protocol                            | n/a
+|--regexes|                   filter flows using regular expression      | n/a
+|--sport_range|               filter source port range                   | n/a
+|--sport|                     filter source port                         | n/a
+|--sports|                    filter on either of two source ports       | n/a
+|--tcp_flags|                 filter TCP flags                           | n/a
+|--interfaces|                interfaces to monitor                      | n/a
 |===
 
 .Example running flows capture on TCP protocol and port 49051 with PacketDrop and RTT features enabled:
 [source,terminal]
 ----
-$ oc netobserv flows --enable_pktdrop=true  --enable_rtt=true --enable_filter=true --action=Accept --cidr=0.0.0.0/0 --protocol=TCP --port=49051
+$ oc netobserv flows --enable_pkt_drop  --enable_rtt --enable_filter --action=Accept --cidr=0.0.0.0/0 --protocol=TCP --port=49051
 ----
-
 [id="cli-reference-packet-capture-options_{context}"]
 == Packets capture options
-You can filter on port and protocol for packet capture data. 
+You can filter packets capture data as same as flows capture using the filters.
+Note that the features, such as packets drop, DNS, RTT, and network events, are only available for flows and metrics capture.
```

Comment:
Could "Note the..." be dropped? Something like:
```suggestion
Certain features, such as packets drop, DNS, RTT, and network events, are only available for flows and metrics capture.
```
or more directly:
```suggestion
The packets drop, DNS, RTT, and network events features are only available for flows and metrics capture.
```

Reply:
I like the former. I'll take it :)